app/graphql/types/api/query_type.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Types
+  module Api
+    class QueryType < Types::BaseObject
+      # Add `node(id: ID!) and `nodes(ids: [ID!]!)`
+      include GraphQL::Types::Relay::HasNodeField
+      include GraphQL::Types::Relay::HasNodesField
+
+      field :current_user, resolver: Resolvers::CurrentUserResolver
+
+      field :add_on, resolver: Resolvers::AddOnResolver
+      field :add_ons, resolver: Resolvers::AddOnsResolver
+      field :billable_metric, resolver: Resolvers::BillableMetricResolver
+      field :billable_metrics, resolver: Resolvers::BillableMetricsResolver
+      field :coupon, resolver: Resolvers::CouponResolver
+      field :coupons, resolver: Resolvers::CouponsResolver
+      field :credit_note, resolver: Resolvers::CreditNoteResolver
+      field :credit_note_estimate, resolver: Resolvers::CreditNotes::EstimateResolver
+      field :credit_notes, resolver: Resolvers::CreditNotesResolver
+      field :current_version, resolver: Resolvers::VersionResolver
+      field :customer, resolver: Resolvers::CustomerResolver
+      field :customer_invoices, resolver: Resolvers::Customers::InvoicesResolver
+      field :customer_portal_invoice_collections, resolver: Resolvers::CustomerPortal::Analytics::InvoiceCollectionsResolver
+      field :customer_portal_invoices, resolver: Resolvers::CustomerPortal::InvoicesResolver
+      field :customer_portal_organization, resolver: Resolvers::CustomerPortal::OrganizationResolver
+      field :customer_portal_overdue_balances, resolver: Resolvers::CustomerPortal::Analytics::OverdueBalancesResolver
+      field :customer_portal_user, resolver: Resolvers::CustomerPortal::CustomerResolver
```

Comment:
Same as previous comments in the mutation side, shouldn't all this customer portal queries be moved to the customer portal query type? (delete them from here)